src/config.rs

```diff
@@ -150,6 +158,16 @@ pub enum DeviceType {
     #[serde(rename = "s_t_b")]
     Stb = 7,
     AudioDongle = 8,
+    GameConsole = 9,
+    CastAudio = 10,
+    CastVideo = 11,
+    Automobile = 12,
+    Smartwatch = 13,
+    Chromebook = 14,
+    UnknownSpotify = 15,
+    CarThing = 16,
+    Observer = 17,
+    HomeThing = 18,
```

Comment:
I'm a bit confused, why we need those numbers in the enum (not only in the changes you introduced, but also above). If you have the time, we could maybe get rid of them? But surely not necessary for this PR to be mergeable.

Reply:
Well, to be honest I wasn't sure why these enum entries have been explicitly assigned a value either. I've done the same to make the code more consistent but now I'm sure this makes no sense (this is the only enum constructed in such way).

I'll push a commit with the values removed in a few minutes.